app/controllers/organizations_controller.rb

```diff
@@ -146,25 +137,52 @@ def set_users_github_organizations
     end
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def set_filter_options
     @sort_modes = Organization.sort_modes
+    @view_modes = Organization.view_modes
+
+    @current_sort_mode = if @sort_modes.dig(params[:sort_by])
```

Comment:
Non-blocking: Just curious why you used `dig` here (and below) and not just `@sort_modes[params[:sort_by]]` or `@sort_modes.keys.include?(params[:sort_by])`

Reply:
Ah....this was me completely forgetting why John often recommended dig to me. Those were cases where we were going multiple levels deep in a hash and one of the keys might not be there (which would otherwise error). That doesn't apply here (so I've gone ahead and replaced it with `.keys.include?`.

Thanks for catching this!